keyboards/handwired/elrgo_s/keymaps/default/config.h

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
This file isn't needed, and should be removed. 

(eg, it uses the default behaviors which don't need to be defined)